media/formats/mp4/box_definitions.cc

```diff
@@ -1759,6 +1757,14 @@ bool AudioSampleEntry::Parse(BoxReader* reader) {
   }
 #endif  // BUILDFLAG(ENABLE_PLATFORM_AC3_EAC3_AUDIO)
 
+#if BUILDFLAG(ENABLE_PLATFORM_IAMF_AUDIO)
+  if (format == FOURCC_IAMF ||
+      (format == FOURCC_ENCA && sinf.format.format == FOURCC_IAMF)) {
+    RCHECK_MEDIA_LOGGED(reader->ReadChild(&iacb), reader->media_log(),
+                        "Failure parsing IamfSpecificBox (iacb)");
+  }
```

Comment:
This is different than the Chrome code.
if (format == FOURCC_IAMF) {
    RCHECK_MEDIA_LOGGED(iacb.Parse(reader), reader->media_log(),
                        "Failure parsing IamfSpecificBox (iacb)");
    return true;
  }

Reply:
I've updated the PR description with the full commit message. The [second commit](https://chromium-review.googlesource.com/c/chromium/src/+/5559111) to Chromium changes this block